mappings/net/minecraft/client/realms/gui/screen/RealmsInfoScrollableWidget.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_qnehaoor net/minecraft/client/realms/gui/screen/RealmsInfoScrollableWidget
+	FIELD f_pmtdrgtz multilineTextWidget Lnet/minecraft/unmapped/C_ivtezidq;
+	METHOD <init> (IIIILnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_mavozmpp;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 height
+		ARG 5 text
+	METHOD m_jclajfmw setTextColor (I)Lnet/minecraft/unmapped/C_qnehaoor;
```

Comment:
```suggestion
	METHOD m_jclajfmw withTextColor (I)Lnet/minecraft/unmapped/C_qnehaoor;
```
since this method returns the class, I'd use the `with` prefix instead